dask_cloudprovider/aws/ecs.py

```diff
@@ -674,7 +674,7 @@ def __init__(
         environment=None,
         tags=None,
         find_address_timeout=None,
-        skip_cleanup=None,
+        skip_cleanup=True,
```

Comment:
This seems like a pretty important change

Reply:
I adjusted skip_cleanup to True because my thinking is that the user should have to opt-in to destructive AWS resource requests rather than have them occur by default.  But let me know your thoughts.